theme_yes/views/snippets/s_image_gallery.xml

```diff
@@ -9,7 +9,7 @@
     </xpath>
     <!-- Shape -->
     <xpath expr="//*[hasclass('container')]" position="before">
-        <div class="o_we_shape o_web_editor_Floats_12 o_we_flip_x o_we_flip_y"/>
+        <div class="o_we_shape o_web_editor_Floats_12" style="background-image: url('/web_editor/shape/web_editor/Floats/12.svg?c1=o-color-1&amp;c2=o-color-2&amp;c3=o-color-3&amp;c5=o-color-5&amp;flip=xy'); background-position: 50% 100%;"/>
```

Comment:
Is there a value in double flipping this one? Just curious but since the shapes is moving points, I wonder... :grimacing:

Reply:
> Is there a value in double flipping this one? Just curious but since the shapes is moving points, I wonder... grimacing

Imo there is no value, I guess it comes from a copy/paste or something...
I'm ok to remove it if you agree

Reply:
Yes, if it does not seem useful at all, let's unflip this :+1: